test_server_environment/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
do we really need a separata package for testing?

Reply:
To have models, I don't like much creating tests models conditionally in the base addon (test addons is what's used in the core so I'm sure this is supported)